myriad-scheduler/src/main/java/com/ebay/myriad/scheduler/fgs/OfferUtils.java

```diff
@@ -0,0 +1,35 @@
+package com.ebay.myriad.scheduler.fgs;
+
+import java.util.Collection;
+import org.apache.hadoop.yarn.api.records.Resource;
+import org.apache.mesos.Protos;
+import org.apache.mesos.Protos.Offer;
+
+/**
+ * Utility class that provides useful methods that deal with Mesos offers.
+ */
+public class OfferUtils {
+
+  /**
+   * Transforms a collection of mesos offers into {@link Resource}.
+   *
+   * @param offers collection of mesos offers
+   * @return a single resource object equivalent to the cumulative sum of mesos offers
+   */
+  public static Resource getYarnResourcesFromMesosOffers(Collection<Offer> offers) {
+    double cpus = 0.0;
+    double mem = 0.0;
+
```

Comment:
Observed that when running Map Reduce tasks, I was unable to get full memory utilization from Mesos Offers.  It looks like this was because I had a high memory to cpu ratio and containers required only 1 GB Ram.  This caused many 1GB/1CPU tasks to be created until no more CPU's were available.  Suggestion: have a configurable mesosToYARNCPURatio, return floor(cpus/mesosToYarnCPURation).